packages/backend/src/migration/1593764167709-alterTables.ts

```diff
@@ -24,6 +28,8 @@ export class alterTables1593764167709 implements MigrationInterface {
         await queryRunner.query(`ALTER TABLE "modos"."user" ALTER COLUMN "id" DROP DEFAULT;`, undefined);
         await queryRunner.query(`DROP SEQUENCE "modos"."user_id_seq";`, undefined);
         await queryRunner.query(`ALTER TABLE "modos"."user" ALTER COLUMN "id" ADD GENERATED ALWAYS AS IDENTITY;`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user" ADD COLUMN "created_at" TIMESTAMP WITH TIME ZONE DEFAULT NOW();`, undefined);
+        await queryRunner.query(`ALTER TABLE "modos"."user" ADD COLUMN "updated_at" TIMESTAMP WITH TIME ZONE DEFAULT NOW();`, undefined);
```

Comment:
If we got a mix of camel and snake case, it the begining of problem. Stick with CamelCase.

Reply:
Fixed in https://github.com/MediaComem/modos/pull/59/commits/f486309c676e9b5b955fd870acae9251c6080455